docs/ibc-relayer.md

```diff
@@ -0,0 +1,488 @@
+# IBC relaying guide
+
+Babylon uses [IBC](https://ibcprotocol.dev/)
+(Inter-Blockchain Communication protocol) to enable cross-chain
+transfer of tokens.
+To support this capability it relies on
+relayers, processes that can be run by anyone which constantly
+scan for outbound packets on one chain and submits these packets
+alongside corresponding proofs on the destination chain.
+
+In this guide, we will explore how to set up and use
+the [Hermes](https://github.com/informalsystems/hermes) relayer to establish
+IBC connections and relay packets between Babylon and other Cosmos SDK chains.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Hermes](#hermes)
+    - [Configuration](#configuration)
+    - [Add relayer wallets](#add-relayer-wallets)
+    - [Verify configuration files](#verify-configuration-files)
+- [Create a connection between 2 chains](#create-a-connection-between-2-chains)
+    - [Create clients](#create-clients)
+    - [Open connection over new clients](#open-connection-over-new-clients)
+    - [Configure channels in Hermes](#configure-channels-in-hermes)
+- [Start the relayer](#start-the-relayer)
+- [Handling Expired/Frozen IBC Clients](#handling-expiredfrozen-ibc-clients)
+- [Helpful Commands](#helpful-commands)
+
+## Prerequisites
+
+Before beginning, ensure you have:
+
+1. Rust installed and configured
+2. Access to RPC and gRPC endpoints for both chains
+3. Wallets funded with native tokens for both chains
+
+For this tutorial, we will be using:
+
+- Babylon's `bbn-test-5` testnet
+- Cosmos Hub's `theta-testnet-001` testnet
```

Comment:
This network is already decommissioned https://github.com/cosmos/chain-registry/blob/master/testnets/cosmoshubtestnet/chain.json. Maybe use another example?